stacktrace.go

```diff
@@ -226,21 +234,42 @@ func isInAppFrame(frame Frame) bool {
 	return true
 }
 
-// Transform `runtime/debug.*T·ptrmethod` into `{ module: runtime/debug, function: *T.ptrmethod }`
-func deconstructFunctionName(name string) (module string, function string) {
-	if idx := strings.LastIndex(name, "."); idx != -1 {
-		module = name[:idx]
-		function = name[idx+1:]
-	}
-	function = strings.Replace(function, "·", ".", -1)
-	return module, function
-}
-
 func callerFunctionName() string {
 	pcs := make([]uintptr, 1)
 	runtime.Callers(3, pcs)
 	callersFrames := runtime.CallersFrames(pcs)
 	callerFrame, _ := callersFrames.Next()
-	_, function := deconstructFunctionName(callerFrame.Function)
-	return function
+	return baseName(callerFrame.Function)
+}
+
+// packageName returns the package part of the symbol name, or the empty string
+// if there is none.
+// It replicates https://golang.org/pkg/debug/gosym/#Sym.PackageName, avoiding a
+// dependency on debug/gosym.
```

Comment:
This is intentional, because:

1. https://go-proverbs.github.io, ["A little copying is better than a little dependency."](https://www.youtube.com/watch?v=PAAkCSZUG1c&t=9m28s)
2. Avoids increasing the size of binaries that include the SDK (see #75)